stripe/src/main/java/com/stripe/android/view/ExpiryDateEditText.java

```diff
@@ -36,6 +39,12 @@ public ExpiryDateEditText(Context context, AttributeSet attrs, int defStyleAttr)
         listenForTextChanges();
     }
 
+    @Override public void onInitializeAccessibilityNodeInfo(AccessibilityNodeInfo info) {
+        super.onInitializeAccessibilityNodeInfo(info);
+        String accLabel = getText().toString() + " " + getResources().getString(R.string.acc_label_expiry_date);
```

Comment:
I think this should be a string resource, because the translation may not be in the same order in every language (i.e. expiration date + "expiration date")